pandas/core/frame.py

```diff
@@ -1982,6 +1982,9 @@ def to_parquet(self, fname, engine='auto', compression='snappy',
             'pyarrow' is unavailable.
         compression : {'snappy', 'gzip', 'brotli', None}, default 'snappy'
             Name of the compression to use. Use ``None`` for no compression.
+        index : bool
```

Comment:
Say "bool, default True"

Reply:
A bit of a nitpicky question, but do you mean

```
bool, default ``True``
```
or 
```
bool, default True
```
?

Difference is pretty much just formatting in the rendered docs, but I tend to be a stickler for convention.

Reply:
Good question.  The instances that I have seen had no double ticks.